.tekton/lib/tests/test_http_utils.py

```
@@ -0,0 +1,203 @@
+
+        status, body = http_request_with_retry(
+            "https://example.com/api", retries=3, backoff=2)
+
+        self.assertEqual(status, 200)
+        self.assertEqual(mock_req.call_count, 2)
+
+    @patch("http_utils.http_request")
+    def test_custom_should_retry(self, mock_req):
+        mock_req.return_value = (418, "teapot")
+
+        status, body = http_request_with_retry(
```

Ruff (0.15.20)

[warning] Unpacked variable `body` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)